### PR TITLE
Create the security log collection

### PR DESCRIPTION
**Pull request by joelgsamuel**
As per the MOJ Cyber Security 'Security Monitoring Strategy' work, this collection has been drafted to define standards for MOJ technology solutions in relation to creating log data for security monitoring purposes.

### CODE REVIEW

**Comment by joelgsamuel** [comment]
Kindly review->merge @cybersquirrel

File: principles/security-log-collection.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## MOJ Cyber Security Logging Platform
+
+The MOJ Cyber Security team operate a centralised, scalable, multi-tenant, cloud-based log collection and forwarding system for infrastructure (non-application level) log data.
+
+The platform can receive, store, index, filter, search, alert and re-forward log data from any MOJ source (including supplier systems).
+
+## Additive technology supply chain
+
+The security log collection principles are designe to be met through technology supply chain as opposed to each system individually.
```
Comment (cybersquirrel) [inline principles/security-log-collection.md]: missing 'd' on designed.

File: standards/custom-applications.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+---
+
+# Custom Applications
+
+We have developed a series of logging requirements for custom applications, such as digital services, applications materially customised that they can reasonably be considered bespoke/custom for the MOJ and line of business applications at different maturity tiers in order to support defenisve cyber security, such as detecting breaches.
+
+## Baseline Maturity Tier
+
+### 1. User directory services
+Log Collection Principle(s): 1, 2
```
Comment (cybersquirrel) [inline standards/custom-applications.md]: A thought for the future maybe - cross linking to the principles?

Comment (joelgsamuel) [reply standards/custom-applications.md]: Yes, relative links throughout would be good in the future.

**Pull request merged**